profiles/common/modules/features/nexteuropa_editorial/includes/nexteuropa_editorial.tracking.inc

```diff
@@ -0,0 +1,429 @@
+  // Generates errors only if required by WYSIWYG tracking workflow settings.
+  if ($tracking_to_check) {
+    $tracking_activate = $values['tracking_activate'];
+
+    // Treating other translation revisions.
+    if ($tracking_activate) {
+      $entity_message = _nexteuropa_editorial_tracking_entity_message($tracking_activate, $current_ent_lang);
+      form_set_error('tracking_activate', $entity_message);
+    }
+  }
+}
+
+/**
+ * A "Text" field validation callback.
+ *
+ * It checks if the field value contains tracked changes.
+ * If it contains ones, it stores the field name in the form_state.
+ * It will be used during the entity validation.
+ *
+ * @param array $element
+ *   Field element that has been edited.
+ * @param array $form_state
+ *   The form_state related to the submitted form.
+ * @param array $form
+ *   The submitted form.
+ *
+ * @see _nexteuropa_editorial_tracking_entity_validate()
+ */
+function _nexteuropa_editorial_tracking_field_validate($element, &$form_state, $form) {
+  $value = '';
+  $field_name = $element['#field_name'];
+  if (isset($element['#value'])) {
+    $value = $element['#value'];
+  }
+  elseif (isset($element['value'])) {
+    // The callback is called twice for field with text format.
+    // In this case, the inserted value is stored differently.
+    $value = $element['value']['#value'];
+    $element = $element['value'];
+  }
+
+  unset($form_state['tracked_changed_detected'][$field_name]);
+  if (preg_match('/<span[^>]+class="[^"]*ice-[^>]+>/', $value)) {
+    // Storing field info in form_state to treat it later with the entity.
+    $form_state['tracked_changed_detected'][$field_name] = $element['#name'];
+  }
+}
+
+/**
+ * Gets the entity level error message for tracked changes are detected.
+ *
+ * @param array $tracking_lang
+ *   List of languages where tracked changes are detected.
+ * @param string $current_lang
+ *   The current language if it must be removed from the message.
+ *
+ * @return string
+ *   The message to display
+ *
+ * @see _nexteuropa_editorial_tracking_entity_validate()
+ */
+function _nexteuropa_editorial_tracking_entity_message($tracking_lang, $current_lang) {
+  if (isset($tracking_lang[LANGUAGE_NONE])) {
+    unset($tracking_lang[LANGUAGE_NONE]);
+    $tracking_lang[$current_lang] = $current_lang;
+  }
+  // We remove LANGUAGE_NONE because it is common to all translations.
+  unset($tracking_lang[LANGUAGE_NONE]);
+
+  $count = count($tracking_lang);
+  switch ($count) {
+    case 0:
+      // LANGUAGE_NONE case.
+      return t('The form cannot be saved because of tracked changes existing in this revision.');
+
+    case 1:
+      $implied_lang = end($tracking_lang);
+      $lang_name = locale_language_name($implied_lang);
+      $message_param = array('@lang' => $lang_name);
+      return t('The form cannot be saved because of tracked changes existing in the @lang version.',
```

Comment:
Similar. Could be:
'Current content includes unprocessed changes in the @lang version. Please accept or reject them in order to save progress.'